docs/monitoring-usage.rst

```diff
@@ -278,3 +278,91 @@ follows::
 
 .. _Time Series:
     https://cloud.google.com/monitoring/api/ref_v3/rest/v3/TimeSeries
+
+
+Writing Custom Metrics
+---------------------------
+
+The Stackdriver Monitoring API can be used to write data points to custom metrics. Please refer to
+the documentation on `Custom Metrics`_ for more information.
+
+To write a data point to a custom metric, you must provide an instance of
+:class:`~gcloud.monitoring.metric.Metric` created with a metric type as well as values for
+the labels associated with that metric type. The metric type can either
+be created using the :class:`~gcloud.monitoring.metric.MetricDescriptor` class or be
+auto-created.  For more information on creating metric descriptors, see the `Metric Descriptors`_
+section. For information on metric type auto-creation, see `Auto-creation of custom metrics`_.
+
+You must also create a fully parameterized :class:`~gcloud.monitoring.resource.Resource` from one
+of the available resource descriptors which can be listed using the
+:class:`~gcloud.monitoring.resource.ResourceDescriptor` class, which similarly requires that its
+available labels have values specified. The ``project_id`` label is a special case that should
+not be specified.
```

Comment:
I am re-writing Ken's suggestion for the paragraph re-write as is - it provides some documentation around picking the appropriate resource type which is useful for a first time user:

You will also need to provide a ``Resource`` instance specifying a monitored resource type as well as values for all of the monitored resource labels, except for ``project_id``, which is ignored when writing. A good choice is to use the underlying physical resource where your application code runs – e.g., a monitored resource type of ``gce_instance`` or ``aws_ec2_instance``. In some limited circumstances, such as when only a single process writes to the custom metric, you may choose to use the global monitored resource type.